internal/gossiphttp/message_test.go

```diff
@@ -0,0 +1,54 @@
+package gossiphttp
+
+import (
+	"bytes"
+	"io"
+	"math"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Fuzz_message ensures that any byte slice encoded as a ckit gossip message
+// can be decoded back to its original form.
+func Fuzz_message(f *testing.F) {
+	tt := []string{"Hello, world"}
+	for _, tc := range tt {
+		f.Add([]byte(tc))
+	}
+
+	f.Fuzz(func(t *testing.T, data []byte) {
+		if len(data) > math.MaxUint16 {
```

Comment:
I know this probably came from me but it would be good to comment why this is happening:

```suggestion
		// Ignore slices longer than the max message length.
		if len(data) > math.MaxUint16 {
```